src/components/select/Select.js

```diff
@@ -363,8 +363,19 @@ class Select extends PureComponent {
   });
 
   render() {
-    const { components, creatable, error, inverse, helpText, size, success, warning, forwardedRef, ...otherProps } =
-      this.props;
+    const {
+      components,
+      noOptionsMessage = 'No options',
```

Comment:
PP: I would drop the default value as it is already the default in `react-select`.

```suggestion
      noOptionsMessage,
```